tests/llm_translation/test_bedrock_completion.py

```diff
@@ -2529,3 +2529,20 @@ def test_bedrock_custom_proxy():
         assert mock_post.call_args.kwargs["url"] == "https://some-api-url/models"
 
         assert mock_post.call_args.kwargs["headers"]["Authorization"] == "Bearer Token"
+
+@pytest.mark.parametrize(
```

Comment:
can you mock these tests ? Would prefer to not have this make an actual API call